modules/features/kids/src/main/java/au/com/shiftyjelly/pocketcasts/kids/KidsBottomSheetDialog.kt

```diff
@@ -106,5 +114,16 @@ class KidsBottomSheetDialog : BottomSheetDialogFragment() {
                 }
             }
         }
+        snackBarCoordinatorLayout = (activity as FragmentHostListener).snackBarView()
```

Comment:
If you declare `snackBarCoordinatorLayout` as nullable type you should use `as?` here instead of `as?. Or make it a non nullable `lateinit var`.